benchmarks/bench_chonkie.py

```
@@ -0,0 +1,232 @@
+    """Check if file is a text file based on MIME type."""
+    return entry['mime'].startswith('text/') or entry['mime'] in (
+        'application/json',
+        'application/xml',
+        'application/javascript',
+    )
+
+
+def parse_document(entry):
+    """Read file content and compute hash."""
+    try:
+        with open(entry['path'], 'r', encoding='utf-8', errors='replace') as f:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)